zadani/soft/index.html

```diff
@@ -1,96 +1,138 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>Cestování po Islandu</title>
-        <link href="https://fonts.googleapis.com/css?family=Merriweather:400,400i|Raleway:400,700" rel="stylesheet">
-        <link rel="stylesheet" href="style.css">
-    </head>
-    <body>
 
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Cestování po Islandu</title>
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:400,400i|Raleway:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
 
-        <!-- hlavička -->
 
+
+<body>
+
+
+
+
+
+
+
+    <!-- hlavička -->
+    <section class="hlavicka">
         <h1 class="titulek">Island</h1>
 
-        <p class="perex">Když se řekne Island, spoustě lidem se jako první vybaví sníh a led. Sama jsem tam strávila téměř rok života a zem pokrytou sněhem a ledem jsem zažila jen jediný den. Jednou věcí si ale jistá jsem: že návštěva ostrova pod polárním kruhem je pro každého tak silným zážitkem, že se sem bude chtít vrátit.</p>
+        <p class="perex">Když se řekne Island, spoustě lidem se jako první vybaví sníh a led. Sama jsem tam strávila téměř rok života a zem pokrytou sněhem a ledem jsem zažila jen jediný den. Jednou věcí si ale jistá jsem: že návštěva ostrova pod polárním kruhem je pro
+            každého tak silným zážitkem, že se sem bude chtít vrátit.</p>
 
         <p class="autor">Jana Zcela Vymyšlená</p>
 
+    </section>
 
+    <div class="obal">
+        <div class="desktopmenu">
 
+            <!-- menu -->
 
-        <!-- menu -->
+            <ul class="menu">
+                <li class="ohraniceni"><a href="#">Úvod</a></li>
+                <li class="ohraniceni"><a href="#">Proč jet na Island</a></li>
+                <li class="ohraniceni"><a href="#">Co musíte vidět</a></li>
```

Comment:
Tady bychom se asi obešli bez třídy `ohraniceni` a rámečky mohli nastavovat jen na položkách seznamu.
Pokud chceme poslední nastylovat jinak, šlo by to i pomocí selektoru `:last-child`, který už taky umíme - https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child 🙂